static/app/components/profiling/FlamegraphTooltip/flamegraphTooltip.tsx

```diff
@@ -22,6 +23,19 @@ export function formatWeightToProfileDuration(
   return `(${Math.round((frame.totalWeight / flamegraph.profile.duration) * 100)}%)`;
 }
 
+function formatFileNameAndLineColumn(frame: FlamegraphFrame): string | null {
+  if (!frame.frame.file) {
+    return null;
```

Comment:
maybe 
```suggestion
    return '<unknown file>;
```
would be better here

Reply:
also, this will never happen because of the `defined(props.frame.frame.file)` below.